.github/workflows/ff-merge.yml

```
@@ -0,0 +1,30 @@
+name: FF merge
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  merge-upstream:
+    runs-on: ubuntu-20.04
```

動作チェック系以外はlatestのが良い気がしました。（追従めんどくさいので。。）
```suggestion
    runs-on: ubuntu-latest
```